src/components/scrim/edit-scrim-form.tsx

```diff
@@ -293,42 +310,64 @@ export function EditScrimForm({
           <FormItem>
             <FormLabel>Maps</FormLabel>
             <Accordion type="single" collapsible className="max-w-lg">
-              {maps.map((map) => (
+              {maps.map((map, index) => (
                 <AccordionItem key={map.id} value={map.id.toString()}>
                   <AccordionTrigger>{map.name}</AccordionTrigger>
                   <AccordionContent>
-                    <AlertDialog>
-                      <AlertDialogTrigger asChild>
-                        <Button variant="destructive">Delete</Button>
-                      </AlertDialogTrigger>
-                      <AlertDialogContent>
-                        <AlertDialogHeader>
-                          <AlertDialogTitle>
-                            Are you absolutely sure?
-                          </AlertDialogTitle>
-                          <AlertDialogDescription>
-                            This will permanently delete the map{" "}
-                            <strong>{map.name}</strong>. This action cannot be
-                            undone.
-                          </AlertDialogDescription>
-                          <AlertDialogFooter>
-                            <AlertDialogCancel>Cancel</AlertDialogCancel>
-                            <AlertDialogAction
-                              onClick={() => deleteMap(map.id)}
-                            >
-                              {loading ? (
-                                <>
-                                  <ReloadIcon className="mr-2 h-4 w-4 animate-spin" />{" "}
-                                  Deleting...
-                                </>
-                              ) : (
-                                "Delete"
-                              )}
-                            </AlertDialogAction>
-                          </AlertDialogFooter>
-                        </AlertDialogHeader>
-                      </AlertDialogContent>
-                    </AlertDialog>
+                    <div className="flex items-center justify-between">
+                      <FormField
+                        control={form.control}
+                        name={`maps.${index}.replayCode`}
+                        render={({ field }) => (
+                          <FormItem>
```

Comment:
Add a `classname="pl-1"` here to avoid clipping:
![CleanShot 2024-08-23 at 00 49 31@2x](https://github.com/user-attachments/assets/c3eee07a-d56e-49e3-8dac-aeb42d82f18c)

After:
![CleanShot 2024-08-23 at 00 50 36@2x](https://github.com/user-attachments/assets/313f7ab2-0609-4360-b5bc-04e2a12d18fc)

Reply:
Should look like this: `<FormItem className="pl-1">`